expense/workflow.py

```diff
@@ -0,0 +1,56 @@
+from datetime import timedelta
+
+from temporalio import workflow
+
+with workflow.unsafe.imports_passed_through():
+    from expense.activities import (
+        create_expense_activity,
+        payment_activity,
+        wait_for_decision_activity,
+    )
+
+
+@workflow.defn
+class SampleExpenseWorkflow:
+    @workflow.run
+    async def run(self, expense_id: str) -> str:
+        logger = workflow.logger
+
+        # Step 1: Create new expense report
+        try:
+            await workflow.execute_activity(
+                create_expense_activity,
+                expense_id,
+                start_to_close_timeout=timedelta(seconds=10),
+            )
+        except Exception as err:
+            logger.error(f"Failed to create expense report: {err}")
```

Comment:
Change to `logger.exception` if you want the exception info like message and stack trace automatically added